src/parsers/utils/assignBackboneHBonds.ts

```diff
@@ -1,15 +1,47 @@
 // This will identify all hydrogen bonds between backbone
 // atoms; assume atom names are correct, only identifies
 // single closest hbond
-export function assignBackboneHBonds(atomsarray: string | any[], hbondCutoff: number) {
-  let maxlength = hbondCutoff || 3.2;
-  let maxlengthSq = maxlength*maxlength;
-  let atoms = [];
+// interface Atoms {index: number; atom: string; hbondDistanceSq: number; hbondOther: any; hetflag:any}
+
+export interface Atom {
```

Comment:
AtomSpec is already defined in spec.ts.  Please do not define a duplicate type.

Reply:
Thank you pointing it out, I was not aware of it mostly because it was not used anywhere in `utils`.
Additionally, I have also added some of the missing types in `AtomSpec`. Currently, all the properties are optional, which shouldn't be the case, but as I further refactor other files and get better types for defining the properties, I'll update them in subsequent commits/PRs